File: Python/MonitorGaming.py
from Monitor import Monitor

class MonitorGaming(Monitor):
    def __init__(self, id=0, nama="", merek="", harga="", gambar="",
                 ukuran_layar="", layar="", resolusi="",
                 refresh_rate="", tambahan_kabel="", mode=""):
        super().__init__(id, nama, merek, harga, gambar,
                         ukuran_layar, layar, resolusi)
        self.refresh_rate = refresh_rate
        self.tambahan_kabel = tambahan_kabel
        self.mode = mode

    def to_list(self):
        """Return data dalam bentuk list untuk tabulate"""
        return [self.id, self.nama, self.merek, self.harga, self.gambar,
                self.ukuran_layar, self.layar, self.resolusi,
                self.refresh_rate, self.tambahan_kabel, self.mode]

File: Java/MonitorGaming.java
public class MonitorGaming extends Monitor {
    private String refreshRate;
    private String tambahanKabel;
    private String mode;

    public MonitorGaming() {
        super();
        this.refreshRate = "";
        this.tambahanKabel = "";
        this.mode = "";
    }

    public MonitorGaming(int id, String nama, String merek, String harga, String gambar,
                         String ukuranLayar, String layar, String resolusi,
                         String refreshRate, String tambahanKabel, String mode) {
        super(id, nama, merek, harga, gambar, ukuranLayar, layar, resolusi);
        this.refreshRate = refreshRate;
        this.tambahanKabel = tambahanKabel;
        this.mode = mode;
    }

    public void tampil() {
        System.out.printf("%-5d %-12s %-10s %-10s %-10s %-12s %-10s %-10s %-12s %-15s %-10s\n",
                id, nama, merek, harga, gambar,
                ukuranLayar, layar, resolusi,
                refreshRate, tambahanKabel, mode);
    }
}
